charts/dify/Chart.yaml

```
@@ -20,13 +20,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.6
+version: 0.4.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.6.9"
+appVersion: "0.6.10"
```

```suggestion
appVersion: "0.6.11"
```
The latest version is https://github.com/langgenius/dify/releases/tag/0.6.11

---

Oh, didn't notice that. Thanks!